map/src/main/kotlin/edu/artic/map/carousel/TourCarouselFragment.kt

```diff
@@ -114,12 +111,14 @@ class TourCarouselFragment : BaseViewModelFragment<TourCarouselViewModel>() {
 
         viewModel.currentPage
                 .distinctUntilChanged()
+                .observeOn(AndroidSchedulers.mainThread())
                 .subscribe { page ->
                     tourCarousel.setCurrentItem(page, true)
                 }
                 .disposedBy(disposeBag)
 
         tourCarousel.pageSelections()
+                .skip(1)
```

Comment:
I take it this is intended to skip the first selection of position `0` (the one emitted during layout)?

Reply:
Replaced `skip` to `skipInitialValue`